package on2013_06.on2013_06_14_Codeforces_Round__188.D___Game_with_Powers;

import net.egork.chelper.tester.NewTester;

import org.junit.Assert;
import org.junit.Test;

public class Main {
	@Test
	public void test() throws Exception {
		if (!NewTester.test("lib/test/on2013_06/on2013_06_14_Codeforces_Round__188/D___Game_with_Powers/D - Game with Powers.task"))
			Assert.fail();
	}
}
